.gitignore

```diff
@@ -31,7 +31,7 @@ moflextab.py
 testsuite/runtests.log
 /testsuite/testtmp/
 /testsuite/schema/mof*/
-/testsuite/end2end/server_file.yml
+server_file.yml
```

Comment:
Change name to something more complete like end2endtest_server_definitions.yml

By the way I can see many more uses fo this file so maybe something even more general

Reply:
The file can also be used by the manual scripts or by pywbemtools tests or by wbemcli tests, so I agree with the broader use. That's why I would not like to have end2end in its name. However, changing server_file to server_definitions sounds good to me. Can we defer that tro a separate update?

Reply:
OK.